nested_lookup/lookup_api.py

```diff
@@ -0,0 +1,62 @@
+from six import iteritems
+from nested_lookup import _nested_lookup
+
+
+def nested_get(document, key):
+    """
+    Method to get the value from a deeply nested document
+    Args:
+        document: Might be List of Dicts (or) Dict of Lists (or)
+         Dict of List of Dicts etc...
+        key: Key to look inside the document
+    Return:
+        Value if found else NULL
+    """
+
+    result = list(_nested_lookup(key=key, document=document))
```

Comment:
Maybe we should create a `set`, if the length of the `set` is `0`, return `None`, if the length is `1` return the `result[0]` and if greater than `1` throw an exception?